org.eclipse.scanning.test/src/org/eclipse/scanning/test/epics/EpicsV4ConnectorTest.java

```diff
@@ -122,14 +129,16 @@ public void connectToHangingService() throws Exception {
 		DeviceRunner runner = new DeviceRunner(EPICSv4EvilDevice.class);
 		epicsv4Device = runner.start();
 
-		try (MalcolmService hanger = new MalcolmService(new HangingGetConnectorService(), null)) {
-	        System.setProperty("org.eclipse.scanning.malcolm.core.timeout", String.valueOf(100));
+		try {
+			System.setProperty("org.eclipse.scanning.malcolm.core.timeout", String.valueOf(100));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "org.eclipse.scanning.malcolm.core.timeout" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)